bootstrap.sh

```diff
@@ -12,29 +12,29 @@ apt-get -qq update
 apt-get -qq install -y curl git htop awscli wget build-essential openssl zip software-properties-common gnupg libssl-dev
 
 echo "Configure AWS SQS access"
-SQS_IDENT=`cat /home/vagrant/sqs.txt`
 
 if [[ "$SQS_IDENT" != _* ]]
 	then
-	echo "ERROR! Missing or empty sqs.txt file!"
-	echo "ERROR! Missing or empty sqs.txt file!"
-	echo "ERROR! Missing or empty sqs.txt file!"
-	echo "ERROR! Missing or empty sqs.txt file!"
-	echo "ERROR! Missing or empty sqs.txt file!"
+	echo "ERROR! Missing or malformed critical environment variables: SQS_IDENT"
 	exit 1
 fi
 if [[ -z "$AWS_REGION" || -z "$AWS_ACCESS_KEY_ID" || -z "$AWS_ACCESS_SECRET" ]]
     then
 	echo "ERROR! Missing critical environment variable: AWS_REGION, AWS_ACCESS_KEY_ID, AWS_ACCESS_SECRET!"
     exit 1
 fi
-mv /home/vagrant/sqs.txt /data/www
+
 echo "local" > /data/www/host.txt
+runuser -l vagrant -c "echo $SQS_IDENT > /data/www/sqs.txt"
```

Comment:
Is this going to do what we want? From `man runuser`:

```quote
       -, -l, --login
              Start the shell as a login shell with an environment similar to a real login:

                 o      clears all the environment variables except for TERM

                 o      initializes the environment variables HOME, SHELL, USER, LOGNAME, PATH

                 o      changes to the target user's home directory

                 o      sets argv[0] of the shell to '-' in order to make the shell a login shell
```

That "clears all the environment variables" bit suggests to me that `$SQS_IDENT` will not survive.

I think probably we want this to match the previous line:

```suggestion
echo $SQS_IDENT > /data/www/sqs.txt
```

I know you're in the process of `vagrant destroy; vagrant up`, so if it works for you as-is then great, no change needed!

Reply:
This was bad for another reason too: this directory is only writeable by root at this point. This is the correct solution, which I realized independently and pushed. Thanks for catching it too though!~